mirrord/operator/src/client.rs

```diff
@@ -642,16 +646,26 @@ impl OperatorApi<PreparedClientCert> {
     where
         P: Progress,
     {
-        self.check_copy_target_feature_support(config)?;
+        self.check_feature_support(config)?;
 
-        let target = if config.feature.copy_target.enabled {
+        let target = if config.feature.copy_target.enabled
+            // use copy_target for splitting queues
```

Comment:
I don't think this should be done transparently. I think if `copy_target` is not enabled it's a config verification error. @aviramha wdyt?

Reply:
Wdyt about a warning then?

Reply:
Ok, let it be `tracing::info!`

Reply:
added.